ggplot_recreate_challenge/exercises.Rmd

```diff
@@ -5,26 +5,670 @@ output:
     df_print: paged
 ---
 
-```{r setup}
+```{r setup, include=FALSE}
+# knit options
+knitr::opts_chunk$set(
+  echo = TRUE,
+  eval = FALSE
+)
+knitr::opts_knit$set(
+  width = 78,
+  progress = FALSE
+)
 # Load your packages!
-
+library(tidyverse)
+library(skimr)
+library(ftplottools)
 ```
 
 # Exercise
 
-The purpose of this exercise is to make a publication-ready plot using your `dplyr` and `ggplot2` skills. Open the journal article "Riddell_Annals_Hom-Sui-Disparities.pdf". Read the abstract and have a look at Figure 3. The data you need is "CDC_Males.csv". You'll have 1 hour to see what you can come up with in groups or individually!
+The purpose of this exercise is to make a publication-ready plot using your `dplyr` and `ggplot2` skills. Open the journal article "2020-07-24-coronavirus-tracked-the-latest-figures-as-countries-start-to-reopen.pdf". 
+
+Read the article and have a look at the figure titled, *Death rates have climbed far above historical averages in many countries that have faced Covid-19 outbreaks*. 
+
+The data you need is "`FtExcessDeaths.csv.csv`". You'll have 1 hour to see what you can come up with in groups or individually!
 
 You got this. Don't worry about making it exactly, try and see how far you can get. You're encouraged to work together if you want to and exchange tips/tricks you figured out. 
 
-I'll leave these here:
+## Import the data
 
-* https://cran.r-project.org/web/packages/ggrepel/vignettes/ggrepel.html
-* http://colorbrewer2.org
+Import the .csv file below into `FtExcessDeaths`.
 
-Hint: You may need to use some `dplyr` skills from the first session
+```{r import-FtExcessDeaths}
+FtExcessDeaths <- read_csv("FtExcessDeaths.csv")
+```
+
+### View the data 
+
+Use `dplyr::glimpse()` and `skimr::skim()` to view the `FtExcessDeaths` data. 
+
+```{r glimpse}
+dplyr::glimpse(x = 
```

Comment:
Not a problem! I've changed these back. I was looking into using something like [styler](https://www.tidyverse.org/blog/2019/11/styler-1-2-0/) or another package to try and streamline this but decided it was easier just to handle it manually.

I changed the style here, but left the `function::argument()` for `skimr` because I wasn't sure if they've seen it before. 

~~~
### View the data 

Use `dplyr::glimpse()` and `skimr::skim()` to view the `FtExcessDeaths` data. 

```{r glimpse}
glimpse(x = #______)
```

```{r skim}
skim(data = # ______)
```
~~~